pytorch_lightning/trainer/training_loop.py

```diff
@@ -880,7 +880,7 @@ def build_train_args(self, batch, batch_idx, opt_idx, hiddens):
     def save_loggers_on_train_batch_end(self):
         # when loggers should save to disk
         should_flush_logs = self.trainer.logger_connector.should_flush_logs
-        if should_flush_logs or self.trainer.fast_dev_run:
+        if should_flush_logs or self.trainer.fast_dev_run is True:
```

Comment:
why are we changing all these to is True?

Reply:
if `fast_dev_run > 1` the using just `self.trainer.fast_dev_run` will be True always and will generate logs at every step. Others suggested that it should follow `log_every_n_steps` in such case. Also not sure whether `fast_dev_run` should generate logs, since this is happening on master if set to `True`.